docs: Add caution about root user credentials (#2113)
aws/copilot-cli: site/content/docs/credentials.md

!!! tip
    We **recommend using a [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)** to store your application's credentials. 

!!! caution

>     You **can not** use AWS account root user credentials for Copilot.

Reviewer: ```suggestion
    You **cannot** use AWS account root user credentials for Copilot.
```

Reviewer 2: I feel like `!!!caution` is a bit excessive to put at the top for this scenario, given that it's discouraged by IAM to use access keys of the root user so I don't think it's very common. I think maybe instead at l.22 we can add:

> !!!note 
    You cannot use the AWS account root user credentials for your application. Please first create an IAM user instead as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html).
